examples/run_Hahn_echo.py

```diff
@@ -91,21 +91,21 @@
     echo_data.set_units("t2", "s")
     fl.next("Raw - time")
     fl.image(
-        echo_data.C.mean("nScans"))
+        echo_data)
     echo_data.reorder("t2", first=False)
     for_plot = echo_data.C
     for_plot.ft('t2',shift=True)
     for_plot.ft(['ph1'], unitary = True)
     fl.next('FTed data')
-    fl.image(for_plot.C.mean("nScans")
```

Comment:
C is extra.  Data is already copied.  Also, what if there is no nScans dimension?

Reply:
The way the data is saved for a hahn echo or any dataset the nScans dimension is created in the ppg as 1 or more depending on the active ini

Reply:
and this is a deleted line